tests/ipsecondary_test.py

```diff
@@ -145,6 +145,40 @@ def test_secondary_ostree_update(uptane_repo, secondary, aktualizr, treehub, sys
     return True
 
 
+@with_treehub()
+@with_uptane_backend()
+@with_director()
+@with_sysroot()
+@with_secondary(start=False, output_logs=False, force_reboot=True)
+@with_aktualizr(start=False, run_mode='once', output_logs=True)
+def test_secondary_ostree_reboot(uptane_repo, secondary, aktualizr, treehub, sysroot, director, **kwargs):
```

Comment:
Do we have a negative test for this? (Same goes for the Primary, actually.) Less important, but just curious.